objects/SetupChecker/script.lua

```diff
@@ -1925,39 +1966,19 @@ function gainSpirit(player)
     end
 
     local count = 0
-    for i = 1,4 do
-        if not options[i] then
+    for i = 1, 4 do
+        if choices[i] == nil or choices[i].spirit == nil then
```

Comment:
when can `choices[i]` be nil? in getGainSpiritChoices you're inserting an empty table right?

Reply:
while i guess it can't hurt to have the nil check, when `getGainSpiritChoices` doesn't have enough spirits it creates an empty table, not a nil entry

Reply:
oh nvm, choices can have nil entries for the first run, i see now